packages/dev/core/src/scene.constants.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Class used to store scene constants
+ */
+export class SceneConstants {

```

Comment:
if we are already reducing clutter, maybe make that a const object and not a class? no need for a class with static members only.

Reply:
sounds good!